test/Rings/mpoly-test.jl

```diff
@@ -50,9 +50,13 @@ end
   f = x^2 + y^2
   g = x^4*y - x*y^3
   I = [f, g]
+  S, (a, b, c) = PolynomialRing(QQ, ["a", "b", "c"])
+  J = ideal(S, [(c^2+1)*(c^3+2)^2, b-c^2])
+
   @test jacobi_ideal(f) == ideal(R, [2*x, 2*y])
   @test jacobi_matrix(f) == matrix(R, 2, 1, [2*x, 2*y])
   @test jacobi_matrix(I) == matrix(R, 2, 2, [2*x, 4*x^3*y-y^3, 2*y, x^4-3*x*y^2])
+  @test gens(radical(J)) == [c^2-b, b^2*c+c^3+2*c^2+2]
```

Comment:
This test is brittle - the underlying algorithm could change to return a different set of generators resp. the same generators in a different order, which would break the test. How about this instead?
```suggestion
  @test gens(radical(J)) == ideal(R, [c^2-b, b^2*c+c^3+2*c^2+2])
```

Reply:
This is true. I will change this test.

Reply:
This test failed (replacing R with S). I changed it to something else